book/data/introduction.md

```diff
@@ -0,0 +1,78 @@
+(data:introduction)=
+# Introduction
+
+In this chapter we'll cover the key aspects we need to know about data for source separation: what do data for source 
+separation look like, relevant datasets, input representations for training models, and, importantly, how to programatically
+generate training and evaluation data to minimize the time we spend data wrangling and maximize the performance we can squeeze
+out of our data.
+
+## Data for music source separation
+
+The inputs and outputs of source separation model look like this:
+
+PLACEHOLDER: image showing mixture --> model --> stems
+
+For this tutorial, we will assume the inputs and outputs were created in the following way:
+1. Each instrument or voice is recorded in isolation into a separate audio track, called a "stem". The stem may be 
+processed with effects such as compression, reverb, etc.
+2. The mixture is obtained by summing the processed stems.
+
+PLACEHOLDER: diagram of simplified mixing process 
+
+```{note}
+This is a simplified view of music creation. In practice, the mixture (musicians refer to this as the *mix*) typically 
+goes through a *mastering* step which includes the application of multiple non-linear transformations to the mixture signal
+to produce the *master*, which is rarely a simple sum of the stems. Nonetheless, this simplified view (no mastering) allows 
+us to train models that produce compelling results, as we shall see throughout this tutorial. 
+```
+
+Since the master is not a linear sum of the stems, for training a source separation model we instead create our own mixture 
+by summing the stems directly to create a linear mixture. When training a source separation model, we provide this mixture 
+as input, the model outputs the estimated stems, and we compare these to the original stems that were used to create the 
+mixture. The difference between the estimated stems and the original stems is used to update the model parameters during 
+training:
+
+PLACEHOLDER: block diagram of training
+
+The difference between the estimated stems and original stems is also used to *evaluate* a trained source separation model,
```

Comment:
We will have covered this briefly in my section. Probably no need to update this now, but something to keep in mind when we're doing our final pass.